app/code/Magento/InventoryApi/Api/GetProductQuantityInStockInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api;
+
+/**
+ * Service which returns Quantity of products available to be sold by Product SKU and Stock Id
+ *
+ * @api
```

Comment:
Btw, we need to add this service to Web API [webapi.xml](https://github.com/magento-engcom/msi/blob/develop/app/code/Magento/InventoryApi/etc/webapi.xml)

in
https://github.com/magento-engcom/msi/blob/develop/app/code/Magento/InventoryApi/etc/webapi.xml
to make it reachable from outside of Magento using "headless" approach

Reply:
Sure, wondering if I can use the `Magento_InventoryApi::stock` ACL for that API method or should create a new one for reading stock data

Reply:
We have open task for refactoring our ACL rules
Generally, I believe that we need to simplify our resource counts.
Single resource `Magento_InventoryApi::stock` is enough for all operation with stock

But in this case need to discuss maybe need to relate on some product resource